task-bot-app/src/main/java/ru/geekbrains/config/BotConfig.java

```diff
@@ -0,0 +1,17 @@
+package ru.geekbrains.config;
+
+import lombok.Data;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@Data
+@PropertySource("application.properties")
+public class BotConfig {
```

Comment:
Необязательно.
1. Поля класса лучше сделать private

Reply:
`    @Value("${bot.name}")
    private String botName;

    @Value("${bot.token}")
    private String token;`

Reply:
[```suggestion
public class BotConfig {
```](https://github.com/sv-smirnov/Telegram_bot/blob/58829ff8f69acfac331227406a07fcf9a7d86ddb/task-bot-app/src/main/java/ru/geekbrains/config/BotConfig.java#L13-L17)